templates/vite/README.md

```diff
@@ -0,0 +1,24 @@
+This is a [Vite](https://vitejs.dev/) project bootstrapped with [`create-fuels`](https://github.com/FuelLabs/fuels-ts/tree/master/packages/create-fuels).
+
+## Getting Started
+
+1. Start the Fuel development server. This server will start a local Fuel node and provide hot-reloading for your smart contracts.
+
+```bash
+npm run fuels:dev
+```
+
+2. Start the Next.js development server.
+
+```bash
+npm run dev
+```
+
+## Deploying to Testnet
+
+To learn how to deploy your Fuel dApp to the testnet, you can follow our [Deploying to Testnet](https://docs.fuel.network/docs/fuels-ts/creating-a-fuel-dapp/deploying-a-dapp-to-testnet/) guide.
```

Comment:
Do we need to update the guide to use Vite?

Reply:
Yes. Think we should do that in a separate PR though in the interest of speed, since the devrel team wants some new features merged soon as well. The deployment steps are very similar (if not exactly the same) for Next and Vite